production/ksonnet/loki/boltdb_shipper.libsonnet

```diff
@@ -70,13 +70,16 @@
     container.mixin.readinessProbe.httpGet.withPath('/ready') +
     container.mixin.readinessProbe.httpGet.withPort($._config.http_listen_port) +
     container.mixin.readinessProbe.withTimeoutSeconds(1) +
-    $.util.resourcesRequests('4', '2Gi')
+    $.util.resourcesRequests($._config.ingester.cpuRequests, $._config.ingester.memoryLimits)
```

Comment:
`$._config.compactor.cpuRequests` and `$._config.compactor.memoryLimits`

Reply:
thanks. i fixed it.